releases/stellar/2025-02-GMP-v1.1.0.md

```diff
@@ -0,0 +1,212 @@
+# Stellar GMP v1.1.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @nbayindirli (<noah@interoplabs.io>)       |
+| **Deployment** | @RiceAndMeet (<steven@interoplabs.io>) |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [v1.1.0 Crates](https://crates.io/crates/stellar-axelar-gateway/1.1.0)
+- [v1.1.0 Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases/tag/stellar-axelar-gateway-v1.1.0)
+
+## Background
+
+- This is the v1.1.0 Stellar GMP release.
+
+Contract changes in the release:
+
+1. Add contractstorage attribute macro, enabling named keys, typed values, storage method specification in the storage layout, storage schema validation, and built-in TTL extension. Use this macro in lieu of Soroban's contracttype macro on storage enums and calling storage on the env.
+2. Add only_owner and only_operator macros, reducing boilerplate for auth-checks and increasing code readability. Use this macro in lieu of directly checking the owner or operator.
+3. Migrate the axelar-operators contract's storage schema key from 'Operators' to 'Operator'.
+4. Add 'stellar/operators.js is-operator', 'stellar/operators.js add-operator', 'stellar/operators.js remove-operator', 'stellar/operators.js collect-fees', 'stellar/operators.js refund', and 'stellar/operators.js execute' commands to interact with the axelar-operators contract.
```

Comment:
Minor changes not worth mentioning. Only document breaking changes, migrations, or new features in the contracts. The scripts usage will be showcased below